package.json

```diff
@@ -7,7 +7,7 @@
     "clean": "rm -Rf dist",
     "phoenix": "rm -Rf node_modules && rm -Rf package-lock.json && npm i",
     "prepack": "npm run clean && npm run build",
-    "test": "sui-test server --pattern \"src/test/**/*Test.js\"",
+    "test": "sui-test server --pattern \"src/test/*Test.js  src/test/**/*Test.js src/test/**/**/*Test.js\"",
```

Comment:
this one is not needed: src/test/**/**/*Test.js

Reply:
I have tested and If I removed only 6 tests are passed of 9.